app/routes/index.tsx

```diff
@@ -220,18 +227,17 @@ export default function App() {
         </div>
       </main>
       {!embed && (
-        <>
-          <a id="discordChatBtn" href="https://discord.com/invite/Bt8PaRTDQC">
-            <Discord />
-          </a>
-
-          <Bottom
-            fetchMore={fetchMoreQuestions}
-            isSingleQuestion={
-              questions.filter((i) => i.questionState != QuestionState.RELATED).length == 1
-            }
-          />
-        </>
+        <a id="discordChatBtn" href="https://discord.com/invite/Bt8PaRTDQC">
+          <Discord />
+        </a>
+      )}
+      {!embed && !queryFromUrl && (
```

Comment:
does this mean that querying in the url will disable the "fetch more" button? Why?

Reply:
The page without button looks nice and clean, but the extra button was very distracting for me, personally 😅 I can bring it back if more peple will want it, but would prefer to keep it out for now:

<img width="1072" alt="Screenshot 2023-09-04 at 10 49 48" src="https://github.com/StampyAI/stampy-ui/assets/1087670/efc647a7-1baa-41a2-816a-7ca203259627">

Reply:
you can get that by setting `more=disabled`. I'm fine either way

Reply:
good point, we want the footer when it's just `queryFromUrl`, but no footer when it's `embed` 😅 so I ended up with a combination of approaches to cover both use cases

code updated